File: archived/w3resource/python/basic/003.py
"""
3. Write a Python program to display the current date and time.
Sample Output :
Current date and time :
2014-07-05 14:34:14
"""
import datetime
current_date = datetime.datetime.now()
formated_date = current_date.strftime("%Y-%m-%d %H:%M:%S")
print(f"Current date and time :\n{formated_date}")

File: archived/w3resource/javascript/basic/003.js
/*
3. Write a JavaScript program to get the current date.
Expected Output :
mm-dd-yyyy, mm/dd/yyyy or dd-mm-yyyy, dd/mm/yyyy
*/
const dateAndTime = new Date().toLocaleString();
let date = dateAndTime.split(', ');
date = date[0].split('/');
const day = date[1] < 10 ? `0${date[1]}` : date[1];
const month = date[0] < 10 ? `0${date[0]}` : date[0];
const year = date[2];
console.info(`${month}-${day}-${year}`);
console.info(`${month}/${day}/${year}`);
console.info(`${day}-${month}-${year}`);
console.info(`${day}/${month}/${year}`);
